bleak/backends/corebluetooth/scanner.py

```diff
@@ -132,6 +132,11 @@ async def get_discovered_devices(self) -> List[BLEDevice]:
                 for u in advertisementData.get("kCBAdvDataServiceUUIDs", [])
             ]
 
+            service_data = {}
+            adv_service_data = advertisementData.get("kCBAdvDataServiceData", [])
+            for u in adv_service_data:
+                service_data[cb_uuid_to_str(u)]= bytes(adv_service_data[u])
```

Comment:
please add a space before the equals to fix the format test failure

Reply:
done